fiona/ogrext2.pyx

```diff
@@ -35,6 +35,10 @@ from cpython cimport PyBytes_FromStringAndSize, PyBytes_AsString
 
 
 log = logging.getLogger("Fiona")
+class NullHandler(logging.Handler):
+    def emit(self, record):
+        pass
+log.addHandler(NullHandler())
```

Comment:
Yes, it belongs. The way I did logging early on was ignorant of how Python's hierarchical logger works and is wrong. In the future we should remove all these `NullHandler` classes, but keep it for now.